src/bp/core/api.ts

```diff
@@ -295,18 +295,18 @@ export class BotpressAPIProvider {
 
 export function createForModule(moduleId: string): Promise<typeof sdk> {
   // return Promise.resolve(<typeof sdk>{})
-  return container.get<BotpressAPIProvider>(TYPES.BotpressAPIProvider).create(`Mod[${moduleId}]`)
+  return container.get<BotpressAPIProvider>(TYPES.BotpressAPIProvider).create(`Mod[${moduleId}]`, moduleId)
```

Comment:
i'd change `owner` by `identity` and make this `module.${moduleId}`, so that we don't end up with weird stuff like `module.hooks` and `module.botpress` ?

Reply:
Good idea, I was finding kind of weird also